ci-operator/config/openshift/builder/openshift-builder-master.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.12
+    tag: golang-1.13
```

Comment:
Please add this to the 4.4, 4.5, and 4.6 versions of this file.

Reply:
additional versions changed @adambkaplan